### DISCUSSION TITLE
Question: What does the warning mean?

### DISCUSSION THREAD

**Discussion by furoxr** [NomicFoundation/hardhat#5815] [General]
I have written some scripts with ignition to interact with contracts deplyoed to bsc testnet. And when I run the command `npx hardhat ignition deploy ....`, the output includes a warning like that:
```
Warning - previously executed futures are not in the module:
- .....
- .....
 ```
 
 I'm not sure what does it mean, should I do something to remove it? Or I shouldn't use ignition to this kind of stuffs?

**Comment by kanej** [comment]
It means that you have run an Ignition deployment based on a module. The state of the deployment has been saved under the `./ignition/deployments/<deployment-id>` folder where `<deployment-id>` will default to `chain-<chainId>` if none is specified.
You have then edited the module editing/removing elements (so removing a deploy of contract `Foo` say, or renaming the contract to `Foo2`) and then rerun the deployment.
Ignition has checked the state of the previous deployment and concluded that previous contract deployments (or previous calls) that were a success are no longer mentioned in your module. It is warning you in case that was unintentional.
You can get rid of the warning by bringing back the previous contract deploys/calls. Or you can clear out the state of the previous deployment if the previous deployment is no longer relevant.

**Reply by furoxr** [reply]
Thank you for your clear explanation~

I'm trying to use Ignition module to write some scripts interacting with deployed contracts. And those scripts don't need to be saved in the deployment. Do I have to delete them manualy or there are some configs I have missed?
